.github/workflows/pr.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         if: needs.plugin-changed.outputs.rust_changed == 'true'
         # todo: modify downstream release.sh to build plugin via utils
         #  perhaps it's time to rename utils or convert all to the utils build
-        run: nix-build -A utils.debug.x86_64.linux-musl.kubectl-openebs --arg incremental false
+        run: nix-build -A utils.debug.x86_64.linux-musl.kubectl-openebs --arg incremental false --tag "${{ github.event.release.tag_name }}"
```

Comment:
I'm confused what we're trying to do here as well, there's no tag here, this is a PR

Reply:
Here probably not required. Ill remove this. This just validates the build